docs/conf.py

```diff
@@ -106,13 +106,13 @@
 
 # Intersphinx Mapping for Links between different Documentations
 intersphinx_mapping = {
-    'python2': ('https://docs.python.org/2', None),
-    'python2.7': ('https://docs.python.org/2.7', None),
-    'python3': ('https://docs.python.org/3', None),
-    'python35': ('https://docs.python.org/3.5', None),
-    'python36': ('https://docs.python.org/3.6', None),
-    'python37': ('https://docs.python.org/3.7', None),
-    'python38': ('https://docs.python.org/3.8', None),
+    "python2": ("https://docs.python.org/2", None),
+    "python2.7": ("https://docs.python.org/2.7", None),
+    "python3": ("https://docs.python.org/3", None),
+    "python35": ("https://docs.python.org/3.5", None),
+    "python36": ("https://docs.python.org/3.6", None),
+    "python37": ("https://docs.python.org/3.7", None),
+    "python38": ("https://docs.python.org/3.8", None),
```

Comment:
I am just curious, why don't we have a mapping for newer version of Python?
Is that not needed anymore?

Reply:
we do need, but for now I will remove conf.py from this pull request.
But will also include a line in the documentation to check it.